shared/video-sdk/develop/spatial-audio/project-implementation/windows.mdx

```diff
@@ -39,7 +39,7 @@ To implement these features in your <Vpl k="CLIENT" />, take the following steps
     You create an instance of `ILocalSpatialAudioEngine` to configure spatial audio and set self and remote user positions. In `AgoraImplementationDlg.h`, add the following to `CAgoraImplementationDlg`:
 
     ```cpp
-    ILocalSpatialAudioEngine localSpatial;
+    ILocalSpatialAudioEngine* localSpatial;
```

Comment:
I suppose you mean the pointer variable , as you can not instantiate the interface (as it contain pure virtual function)